types/amazon-cognito-auth-js/tslint.json

```diff
@@ -0,0 +1,9 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "interface-name": [
```

Comment:
Please remove these custom rules since this is new package, you want to only extend the dtslint/dt.json and fix any other errors.

Reply:
I removed the custom rule and removed the "I" prefix from all interface names. This revealed the main test file did not explicitly reference any interface types; I added coverage for all the interface types.